CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- [cygnus-common][pSQL] Allow use null for ST_GeomFromGeoJSON
```

Comment:
pSQL referst to "PostgreSQL"? Or is a typo?

I guess the later, given that the change is in a common place to MySQL and PG (NGSIUtils.java), but I'm not 100% sure...

Reply:
pSQL = postgreSQL and postgis